### ISSUE TITLE
Add fatal error handling to `ignite-cli new`

### ISSUE THREAD

**Issue by joshuayoes** [infinitered/ignite#2137]
Currently, when an error gets thrown on `npx ignite-cli new`, it will throw the fatal error.

It would be nice to display the error to the user, along with the following information
- A CLI command with the options they used
- A link to the Ignite github issues template
- A suggestion to run `npx ignite-cli doctor`

**Comment by jamonholmgren** [comment]
🤔 I wonder if there's a way to detect similar issues based on the errors generated and suggest issues to look at. (Just TOL, for future)

**Comment by frankcalise** [comment]
This issue is closely related to https://github.com/infinitered/ignite/issues/2509

**Comment by frankcalise** [comment]
@jvm-cognition With your other fixes to #2539, would you mind adding a link to the console output to the bug template? I think that will cover this issue as well. We can output a message along the lines of:

```
Consider opening an issue with the following information at:
https://github.com/infinitered/ignite/issues/new?template=bug_report.yml&labels=bug
```
